moonbeam-types-bundle/index.ts

```diff
@@ -190,6 +190,17 @@ const TYPES_37_42: RegistryTypes = {
   ...TYPES_36_36,
   AccountId32: "H256",
   AuthorId: "AccountId32",
+  ProxyType: {
```

Comment:
I don't think those existed at that time, but I don't think it would harm

Reply:
https://github.com/PureStake/moonbeam/pull/423/files
Yes youre right "Balances" and "AuthorMapping" were added later. Lets keep this for now out of simplicity